util/missingcrypto.txt

```diff
@@ -476,36 +476,128 @@ EC_KEY_set_default_method(3)
 EC_curve_nid2nist(3)
 EC_curve_nist2nid(3)
 EDIPARTYNAME_it(3)
-ENGINE_get_EC(3)
+ENGINE_add(3)
```

Comment:
We should have a manual page that talks about removing engines that will refer all the ENGINE_ functions and the other functions that get removed/replaced with stubs. It does not need to show the prototypes. It should probably talk about the OPENSSL_ENGINE_STUB macro. All these missing manpages should be replaced with this manual page so the additions to missingcrypto.txt should not be needed.